docs/components/sensor/sensirion-sht3xd.md

```diff
@@ -79,8 +78,7 @@ The following attributes are available for `sensirion-sht3xd` sensors:
 <!-- prettier-ignore -->
 | Attribute | Type | Inclusion | Description |
 | --------- | ---- | --------- | ----------- |
-| `board` | string | **Required** | The `name` of the [board](/components/board/) the sensor is wired to. |
-| `i2c_bus` | string | **Required** | The `name` of the [I2C bus](/components/board/#i2cs) on the board that the sensor is wired to. |
+| `i2c_bus` | string | **Required** | The index of the [I2C bus](/components/board/#i2cs) on the board that the sensor is wired to. |
```

Comment:
@penguinland Is this correct/the best way of describing it what this attribute should be for users? If there's a better way to point them LMK and I can update